docs/libcudacxx/extended_api/math.rst

```diff
@@ -0,0 +1,48 @@
+.. _libcudacxx-extended-api-math:
+
+Math
+=====
+
+.. code:: cuda
+
+   template <typename T>
+   [[nodiscard]] __host__ __device__ constexpr T ceil_div(T a, T b) noexcept;
+
+ceil_div
+---------
+
+- _Requires_: `is_integral_v<T>` is true.
+- _Preconditions_: `a >= 0` is true and `b > 0` is true.
+- _Returns_: divides `a` by `b` and rounds the result up to a multiple of `a`.
```

Comment:
Critical: the specification seems incorrect, since the division result is not rounded up to `a` (or `b`).
```suggestion
- _Returns_: divides `a` by `b` and rounds the result up to an integer.
```

Reply:
I did a slightly more descriptive wording:
- _Returns_: divides `a` by `b`. If `a` is not a multiple of `b` rounds the result up to the next integer value.

Reply:
Superb!